### ISSUE TITLE
With the new UI, staking more tokens requires more steps

### ISSUE THREAD

**Issue by 3venturesio** [eco-stake/restake#673]
With the previous UI, adding more tokens to the actual stake was pretty easy by using the drop down and 'delegating more', now, it requires to open the validator information, clicking on the name, then selecting delegate, and then doing the delegation.

I think it was clearer when it was provided direct link in the drop-down menu

**Comment by tombeynon** [comment]
Agreed - I'll be re-adding Delegate, Redelegate and Undelegate links back into the actions menu ASAP.
